config/initializers/paperclip.rb

```diff
@@ -1,6 +1,6 @@
 Paperclip::Attachment.default_options.merge!(
   path: ":rails_root/uploads/:class/:attachment/:id_partition/:style/:filename",
-  url: "/uploads/:class/:id/:style"
+  url: "/:locale/m/:mission/:class/:id/:style"
```

Comment:
I'm not sure if we need `:locale` in here. Answers are not locale dependent. What was your reasoning for including it?